tensorflow/python/keras/_impl/keras/datasets/cifar10.py

```diff
@@ -27,15 +27,22 @@
 from tensorflow.python.keras._impl.keras.utils.data_utils import get_file
 
 
-def load_data():
+def load_data(fname=None):
   """Loads CIFAR10 dataset.
 
+  Arguments:
+    fname: Name of the file. If an absolute path /path/to/file.txt is
```

Comment:
I think "file_path" would be better here.

Reply:
or "fpath"?

Reply:
Actually, I have considered that name. Now, the `fname` comes from the [get_file()](https://github.com/fchollet/keras/blob/master/keras/utils/data_utils.py#L121) function defined in Keras.